Library/Formula/iojs.rb

```diff
@@ -1,4 +1,8 @@
+require "language/javascript"
+
 class Iojs < Formula
+  include Language::JS
```

Comment:
@mikemcquaid Can you give me an example of what you were thinking?   My knowledge of ruby's class system is failing me.  I gave it a shot here: https://github.com/bcomnes/homebrew/commit/53b7c47ba8c869ea58643e4b11479f0633674fcd

It works, but it seems like this way loses the scope of the formula its called in and I have to pass in all local variables and utility functions.  Any ideas?